nb2kg/nb2kg/managers.py

```diff
@@ -206,11 +216,10 @@ def restart_kernel(self, kernel_id, now=False, **kwargs):
         self.log.info("Request restart kernel: %s", kernel_id)
         kernel_url = self._kernel_id_to_url(kernel_id) + '/restart'
         self.log.info("Request restart kernel at: %s", kernel_url)
-        response = yield fetch_kg(
-            kernel_url,
-            method='POST',
-            body=json_encode({})
-        )
+        # Here 10 is used as the multiplier to increase the request timeout limit for kernel restart.
```

Comment:
Why are we multiplying by 10 when a timeout parameter was explicitly specified? How about we default the value to REQUEST_TIMEOUT * 10, but if provided we use the provided value?

Reply:
The "multiplier-effect" was based on results from another project - which used 12 as the factor.  It is only applied to the _restart_kernel_ and _interrupt_kernel_ requests.  I suppose they found those requests were substantially longer than others - but I've always found them to complete in a timely manner.  That said, the request that takes the longest to complete - by far - is _start_kernel_.  Since that method doesn't use a multiplier at all, I'm inclined to remove the multipliers altogether - which is how its always been with this one exception.  (I suppose _restart_kernel_ should take longer since it encompasses a stop and a start.  I just haven't analyzed many logs that include restart requests.)

If we take the position of having the multiplier in effect by default, then you're looking at a default of 200 seconds - which is way too long to wait should something actually go wrong.  So, I suppose I'm changing my previous approval a bit and arguing that we remove the multipliers since I don't see any evidence that either of those two calls (interrupts being the one I've spent the most time with) requiring multipliers.

Reply:
Makes sense to me. Will change the multiplier for the next push.

Thanks @kevin-bates @lresende